src/Microsoft.DotNet.ServiceFabric.ServiceHost/EnvironmentConfigMapper.cs

```diff
@@ -17,20 +17,20 @@ public EnvironmentConfigMapper(IServiceConfig config)
             _env = Environment.GetEnvironmentVariable("ENVIRONMENT_NAME") ?? "Local";
         }
 
-        public IServiceConfigSection this[string name] => new Section(_config[name], _config[$"{name}-{_env}"]);
+        public IServiceConfigSection? this[string name] => new Section(_config[name], _config[$"{name}-{_env}"]);
```

Comment:
Why is this ?. It literally returns a new thing?

Reply:
I hate nullable so much.  I really don't want us to use it, and I don't like that you are spreading it all over the code base with an unrelated change.

Reply:
This is ? because the interface is, i can remove it though.